### ISSUE TITLE
question of batch size

### ISSUE THREAD

**Issue by zhenwuweihe** [OpenDriveLab/RoboDual#4]
The batch size of the training calvin dataset in the paper is 2048, but I found that on my A100, the maximum batch size can only be 8. What is the reason?

![Image](https://github.com/user-attachments/assets/06314287-86d6-4b6c-acd5-96da7b9544d6)

**Comment by retsuh-bqw** [comment]
An equivalent global batch size of 2048 can be attained by utilizing multiple GPUs in conjunction with gradient accumulation steps.
